practice/practice-02/src/index.pug

```diff
@@ -1,8 +1,25 @@
+include nav-item
+
 doctype html
 html(lang="en")
   head
     meta(charset="UTF-8")
     meta(name="viewport", content="width=device-width, initial-scale=1.0")
     link(rel="stylesheet", href="./sass/main.scss")
     title Food Hut
-  body 
\ No newline at end of file
+  body
+    header.header
+      .logo-wrapper 
+        a(href="javascript:void(0)") 
+          img(src="./assets/images/Logo.png", alt="Logo image").logo
+      .header-content
+        nav.nav
+          input(type="checkbox").toggle-menu
+          .icon-menu 
+          ul.nav-list
```

Comment:
Why did we have navItem instead of navMenu?

Reply:
I don't know what you mean, can you explain more?

Reply:
The purpose of component is making your code more simple. That means instead of generating nav items on index, I think we need include component like navMenu is enough

Reply:
I got it and fixed them.